client-v2/src/components/FrontsEdit/FrontSection.tsx

```diff
@@ -119,6 +120,7 @@ type FrontsComponentProps = FrontsContainerProps & {
     ) => void;
     setFrontHiddenState: (id: string, hidden: boolean) => void;
   };
+  isEditions: boolean;
```

Comment:
Is this used in the component? It's new to the props, but I can't see it in render().